tests/functional/test_selection/fixtures.py

```diff
@@ -84,14 +82,3 @@ def models():
         "model_b.sql": models__model_b_sql,
         "model_a.sql": models__model_a_sql,
     }
-
-
-@pytest.fixture(scope="class")
-def project_files(
-    project_root,
-    tests,
-    models,
-    selectors_yml,
-):
-    write_project_files(project_root, "tests", tests)
-    write_project_files(project_root, "models", models)
```

Comment:
Were we able to remove this because of the existing more widely available `project_files` satisfied the relevant tests? I'm trying to understand if this is extraneous cleanup or if it was necessary for some reason in regards to the happy path testing we are setting up in this branch.

Reply:
Yes, they are covered by the project_files.
And the reason they are being removed is they failed to write certain files(like dbt_project.yml), they might not be relevant anymore with the order of writing project files but I think it is still good to remove.